### DISCUSSION TITLE
addon-viewport: Is there a way to set a rotated defaultViewport?

### DISCUSSION THREAD

**Discussion by theinterned** [storybookjs/storybook#11134] [Help]
I have a story that I would like to render in a landscape iPhone using the `defaultViewport` story parameter. Is there a way to do this?

Basically I would like the story to default to rendering like this:

<img width="230" alt="Screenshot 2020-06-11 15 49 11" src="https://user-images.githubusercontent.com/2694/84432651-1c68e980-abfb-11ea-816b-d1938655948b.png">

**Comment by ndelangen** [comment]
The initial state is hard-coded right now:
https://github.com/storybookjs/storybook/blob/341fc2d90d4f3ae42fd9b68173b9f7b1856917f0/addons/viewport/src/Tool.tsx#L134
